gremlin-go/README.md

```diff
@@ -114,15 +156,19 @@ func main() {
 
 Note: The exact import name as well as the module prefix for `NewDriverRemoteConnection` may change in the future.
 
+### Design Architecture
+
+See [Gremlin-Go Design Overview](design.MD)
+
 # Go Gremlin Language Variant
 
 [Apache TinkerPop™][tk] is a graph computing framework for both graph databases (OLTP) and graph analytic systems
 (OLAP). [Gremlin][gremlin] is the graph traversal language of TinkerPop. It can be described as a functional,
 data-flow language that enables users to succinctly express complex traversals on (or queries of) their application's
 property graph.
 
-Gremlin-Go implements Gremlin within the Go language and can be used on any Go runtime greater than v1.17. One important distinction with Go and Java is that 
-the functions are capitalized, as is required to export functions is Go. 
+Gremlin-Go implements Gremlin within the Go language and can be used on any Go runtime greater than v1.17. One
+important distinction with Go and Java is that the functions are capitalized, as is required to export functions is Go. 
```

Comment:
s/is/in 

perhaps say "one important difference between Go and Java is that"

Reply:
Will update, thanks!